app/src/main/java/com/conupods/MapsActivity.java

```diff
@@ -15,6 +19,15 @@
 
     private GoogleMap mMap;
 
+    // These could to be moved outside of the file
+    private final double SGW_LAT = 45.496080;
+    private final double SGW_LNG = -73.577957;
+    private final double LOY_LAT = 45.458333;
+    private final double LOY_LNG = -73.640450;
+
```

Comment:
If they are final, they should be public

Reply:
Good point. I'll change that if we don't end up moving them to a separate file